### DISCUSSION TITLE
Invoice customer name change

### DISCUSSION THREAD

**Discussion by chrisonline** [kimai/kimai#3860] [Questions & Answers] [answered]
Is there a way to change data from an already created invoice?
I have created an invoice, and now I need to change the customer name a little bit.

**Comment by kevinpapst** [comment, marked as answer]
You can delete the invoice (you have to activate a permission for that) and re-create the invoice.
But you cannot change an invoice. 

See https://www.kimai.org/documentation/invoices.html#deleting-invoices

**Reply by chrisonline** [reply]
OK thanks didn't know that there is a permission for delete.
